boards/arm/qemu_cortex_a9/board.cmake

```diff
@@ -18,4 +18,8 @@ set(QEMU_KERNEL_OPTION
   "-device;loader,file=\$<TARGET_FILE:\${logical_target_for_zephyr_elf}>,cpu-num=0"
   )
 
-board_set_debugger_ifnset(qemu)
+board_runner_args(qemu "--commander=qemu-system-xilinx-aarch64" "--cpu=cortex-a9" "--machine=arm-generic-fdt-7series"
+                  "--dtb=${ZEPHYR_BASE}/boards/${ARCH}/${BOARD}/fdt-zynq7000s.dtb"
+                  "--qemu_option=${ZEPHYR_BASE}/boards/${ARCH}/${BOARD}/qemu_cortex_a9_option.yaml")
+
```

Comment:
Please use CMAKE_CURRENT_LIST_DIR instead of hard-coding these paths.

https://cmake.org/cmake/help/latest/variable/CMAKE_CURRENT_LIST_DIR.html

Please do the same elsewhere in this PR where the same change applies.

Reply:
this has fixed. @mbolivar-ampere